spec/filters/dns_spec.rb

```diff
@@ -72,6 +72,27 @@
         insist { subject.get("foo")[1] } == "carrera.databits.net"
       end
     end
+    
+    describe "dns reverse lookup, non-string field" do
+      let(:plugin) { ::LogStash::Filters::DNS.new("reverse" => "foo") }
+      let(:event) { ::LogStash::Event.new("foo" => {"ip" => "1.2.3.4"} ) }
+
+      before do
+        plugin.register
+        allow(plugin.logger).to receive(:warn).with(any_args)
+      end
+
+      it "should not throw an error when filtering" do
```

Comment:
Im still confusing when using _suggestive_ & _imperative_ forms in test cases. Maybe `it "plugin filter doesn't raise an error"` according to our [ref best practice guide](https://github.com/rubocop/rspec-style-guide#should-in-example-docstrings)?

Reply:
You are right.

```suggestion
      it "does not throw an error when filtering" do
```